### PR TITLE
Create CODEOWNERS file

### PR DESCRIPTION
**Pull request by vam-google**
https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2386

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+google-cloud-dns/*                @garrettjonesgoogle
+google-cloud-errorreporting/*     @garrettjonesgoogle
+google-cloud-examples/*           @garrettjonesgoogle
+google-cloud-language/*           @garrettjonesgoogle
+google-cloud-logging/*            @garrettjonesgoogle
+google-cloud-monitoring/*         @garrettjonesgoogle
+google-cloud-pubsub/*             @pongad
+google-cloud-resourcemanager/*    @garrettjonesgoogle
+google-cloud-spanner/*            @vam
+google-cloud-speech/*             @shinfan
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Maybe choose someone else

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+google-cloud-storage/*            @garrettjonesgoogle
+google-cloud-testing/*            @vam
+google-cloud-trace/*              @garrettjonesgoogle
+google-cloud-translate/*          @garrettjonesgoogle
+google-cloud-util/*               @garrettjonesgoogle
+google-cloud-video-intelligence/* @shinfan
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Maybe choose someone else

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+google-cloud-core-http/*          @garrettjonesgoogle
+google-cloud-core/*               @garrettjonesgoogle
+google-cloud-datastore/*          @garrettjonesgoogle
+google-cloud-dep-verification/*   @garrettjonesgoogle
+google-cloud-dlp/*                @neozwu
+google-cloud-dns/*                @garrettjonesgoogle
+google-cloud-errorreporting/*     @garrettjonesgoogle
+google-cloud-examples/*           @garrettjonesgoogle
+google-cloud-language/*           @garrettjonesgoogle
+google-cloud-logging/*            @garrettjonesgoogle
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Add @jabubake

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+google-cloud-bigquery/*           @vam
+google-cloud-compute/*            @garrettjonesgoogle
+google-cloud-contrib/*            @garrettjonesgoogle
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Add @jean-philippe-martin

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+google-cloud-bigquery/*           @vam
```
Comment (garrettjonesgoogle) [inline .github/CODEOWNERS]: Add @tswast

**Comment by shinfan** [comment]
@vam-google Feel free to change my ownership to @garrettjonesgoogle (as the default owner)

**Comment by vam-google** [comment]
PTAL

**Comment by pongad** [comment]
When committing, rename 'crate' to 'create'. Otherwise, LGTM for pubsub

**Comment by garrettjonesgoogle** [comment]
LGTM

**Comment by tswast** [comment]
LGTM

**Pull request merged**